task3/Task3-1-2/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Text;
+
+namespace Task3_1_2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            PrintAGreating();
+            var analyzer = new TextAnalyzer(ArticleInput());
+            analyzer.InitializeDictionary();
+            var exit = false;
+            int inp;
+            while (!exit)
+            {
+                PrintMenu();
+                while (!int.TryParse(Console.ReadLine(), out inp) || inp > 3 || inp < 1)
+                {
+                    Console.WriteLine("Enter a number between 1 and 3:");
+                }
+                switch (inp)
+                {
+                    case 1:
+                        Console.Clear();
+                        analyzer.ShowFullAnalysis();
+                        break;
+                    case 2:
+                        Console.Clear();
+                        analyzer.ShowShortAnalysis();
+                        break;
+                    case 3:
+                        exit = true;
+                        break;
+                }
+            }
+        }
+
+        private static string ArticleInput()
+        {
+            Console.WriteLine("enter text:");
+            string str;
+            var sb = new StringBuilder();
+            var escape = false;
+            while (!escape)
+            {
+                str = Console.ReadLine();
+                if (str != "stop")
+                {
+                    sb.Append(str + " ");
+                }
+                else
+                {
+                    escape = true;
+                }
+            }
```

Comment:
```suggestion
            while (true)
            {
                str = Console.ReadLine();
                
                if (str == "stop") break;
                
                sb.Append(str + " ");
            }
```
I know that I previously said that while(true) is a tricky thing, but you actually can use it (now) if you are sure that everything is clear. Here `while` is pretty small and `break` is not so far/deep, so you can just use it instead